cypress/integration/website.spec.tsx

```diff
@@ -3,3 +3,5 @@ describe('Dashboard', () => {
         cy.visit('/');
     });
 });
+
+export {};
```

Comment:
0d8e3ead368182625fc060808ed0ea1cb0687cc6

This can also be achieved without exporting by updating the `tsconfig` to support `isolatedModules`.

Source: https://www.typescriptlang.org/tsconfig#isolatedModules